src/components/Pagination/Pagination.tsx

```diff
@@ -315,6 +314,7 @@ export const Pagination: FC<PaginationProps> = React.forwardRef(
                                     <TextInput
                                         ref={inputRef}
                                         classNames={styles.editor}
+                                        shape={TextInputShape.Pill}
```

Comment:
Your commit message and this change seem to be out of sync. changing default to pill?

Reply:
commit message says changing in input and textarea,

Reply:
Ah okay. Thanks for the clarification... I thought pill shape was the desired look in 2.x styling?

Reply:
most places where we are using inputs we have been asked to change to rectangle shape by deisgn, so it makes sense to make it the default

Reply:
Sounds good. I'll defer to Dylan on this call.

Reply:
Will approve for next week's release - need to inform TM, PCS designers and engineering teams about this change. Project X default is Pill, but I see no problem with being discreet about using it.